From ff0dc52fccf69ee441b7df0232c4aadf440fe3ff Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Wed, 10 Feb 2021 19:28:13 -0600
Subject: [PATCH 18/50] Correct documentation.

This update modifies the documentation for the two functions
that generate the measurement of the actor and subject involved
in a event.  This was missed when the function names were
modified.
---
 security/integrity/ima/ima_template_lib.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/security/integrity/ima/ima_template_lib.c b/security/integrity/ima/ima_template_lib.c
index 192555f7aeea..5bfb88df1aa6 100644
--- a/security/integrity/ima/ima_template_lib.c
+++ b/security/integrity/ima/ima_template_lib.c
@@ -470,7 +470,7 @@ int ima_eventmodsig_init(struct ima_event_data *event_data,
 }
 
 /*
- *  ima_event_actor_init - Generate identity of actor involved in event.
+ *  ima_event_actor_init - Generate measurement of event actor.
  */
 int ima_event_actor_init(struct ima_event_data *event_data,
 			 struct ima_field_data *field_data)
@@ -489,7 +489,7 @@ int ima_event_actor_init(struct ima_event_data *event_data,
 }
 
 /*
- *  ima_event_subject_init - Generate identity of subject involved in event.
+ *  ima_event_actor_init - Generate measurement of event subject.
  */
 int ima_event_subject_init(struct ima_event_data *event_data,
 			   struct ima_field_data *field_data)
-- 
2.31.1

